### PR TITLE
Dropped support for Openflow 1.0

### LINKED ISSUE #50: Drop OpenFlow 0x01 support
This is for no longer supporting OpenFlow 1.0.

On this NApp, it should stop listening the OpenFlow 1.0 related events and remove the associated code, settings.py `OPENFLOW_VERSIONS` and updating the README accordingly.

### PR DESCRIPTION
**Pull request by Alopalao**
Issue: Closes #50
I deleted this, is it correct? Looks to me is testing communication of stats from OF 1.0 and 1.3 -> [test_main.py method](https://github.com/Alopalao/of_core/blob/851f3e0646368fa7853c9b786bfaac62069f521f/tests/unit/test_main.py#L208-L257)

### CODE REVIEW

**Comment by Alopalao** [comment]
[flow_stats](https://github.com/kytos-ng/flow_stats/blob/b0712de5128ed43a83fc83b1396b42c915936160/main.py#L19) NApp is affected by this PR

File: tests/unit/test_match.py [hunk 1]
```diff
@@ -88,7 +86,7 @@ def test_match_tlv(self):
         )
 
     def test_match01_as_dict(self) -> None:
-        """Test match01 as_dict."""
+        """Test match04 as_dict."""
```
Comment (viniarck) [inline tests/unit/test_match.py:89]: Aldo, could you update the name of the test as `test_match04_as_dict` as well?

File: tests/unit/test_main.py [hunk 2]
```diff
@@ -205,57 +204,6 @@ async def test_handle_multipart_reply(
         await napp._handle_multipart_reply(ofpmp_desc, switch_one)
         assert switch_one.update_description.call_count == 1
 
-    @patch('napps.kytos.of_core.main.log')
-    @patch('kytos.core.buffers.KytosEventBuffer.aput')
-    @patch('napps.kytos.of_core.main.Main._update_switch_flows')
-    @patch('napps.kytos.of_core.v0x04.flow.Flow.from_of_flow_stats')
-    @patch('napps.kytos.of_core.main.Main._is_multipart_reply_ours')
-    async def test_on_multipart_flow_stats(
```
Comment (viniarck) [inline tests/unit/test_main.py:213]: @Alopalao answering the question you've asked on this PR description. This `test_on_multipart_flow_stats` we want to keep it, this is testing out `_handle_multipart_flow_stats` which is related to OF1.3 (0x04), so, could you revert the deletion of test case?

File: tests/unit/test_match.py (hunk 1 above)
Comment (Alopalao) [reply tests/unit/test_match.py:89]: Yes, I totally missed that.

File: tests/unit/test_main.py (hunk 2 above)
Comment (Alopalao) [reply tests/unit/test_main.py:213]: Got it.

**Review by viniarck — COMMENTED** [review]
@Alopalao, excellent PR. It's almost there, the major part is complete. I've explored it locally to help to try this PR out, looks really great. Other than that, I've linked some related issues/enhancements, if you could also help out with that when you have the chance in addition to the one on flow_stats that you're already working on that'd be awesome, appreciated your help with this.

**Comment by viniarck** [comment]
I forgot to mention, later on, please update the changelog,thanks.

**Review by viniarck — APPROVED** [review]
Nicely done @Alopalao, and thanks for fixing the linked issues. I'll leave this PR pre-approved. 

Before getting it merged, since `of_core` is the most critical one (for this epic that is being completed), let's make sure e2e tests are passing as well, ping me if you need help with that, I think you can select only a subset of the test cases since we know upfront that OF1.0 isn't being tested there, for instance, you can select `flow_manager` related tests with this expression by locally editing `kytos-init.sh` from the e2e tests repo:

```
diff --git a/kytos-init.sh b/kytos-init.sh
index 7f4a19a..3d7c83e 100755
--- a/kytos-init.sh
+++ b/kytos-init.sh
@@ -20,6 +20,8 @@ sed -i 's/WARNING/INFO/g' /etc/kytos/logging.ini
 test -z "$TESTS" && TESTS=tests/

 python3 scripts/wait_for_mongo.py 2>/dev/null
+python3 -m pytest --timeout=600 $TESTS -k flow_manager
+tail -f
 python3 -m pytest $TESTS

 #tail -f
```

**Comment by Alopalao** [comment]
Results from [kytos-end-to-end-tests](https://github.com/amlight/kytos-end-to-end-tests/tree/master/tests) (e2e):
```
tests/test_e2e_01_kytos_startup.py ..                                                                        [  1%]
tests/test_e2e_05_topology.py ..................                                                                [ 10%]
tests/test_e2e_10_mef_eline.py .........Xss.....x....x.....                                               [ 24%]
tests/test_e2e_11_mef_eline.py .....                                                                           [ 26%]
tests/test_e2e_12_mef_eline.py .....xx.                                                                      [ 30%]
tests/test_e2e_13_mef_eline.py .....xxXx......xxXx.XXxX.xxxx..x.E..........                  [ 52%]
tests/test_e2e_14_mef_eline.py x                                                                              [ 53%]
tests/test_e2e_15_maintenance.py ........................                                                   [ 65%]
tests/test_e2e_20_flow_manager.py ................                                                         [ 73%]
tests/test_e2e_21_flow_manager.py ..                                                                       [ 74%]
tests/test_e2e_22_flow_manager.py ...............                                                          [ 81%]
tests/test_e2e_23_flow_manager.py X......................                                                [ 93%]
tests/test_e2e_30_of_lldp.py ....                                                                                [ 95%]
tests/test_e2e_31_of_lldp.py ...                                                                                 [ 96%]
tests/test_e2e_32_of_lldp.py ...                                                                                 [ 98%]
tests/test_e2e_40_sdntrace.py X.x                                                                            [100%]
```
Summary: `170 passed, 2 skipped, 18 xfailed, 8 xpassed, 724 warnings, 1 error`
Error: `Exception: Timeout while starting Kytos controller.` [from wait_controller_start()](https://github.com/amlight/kytos-end-to-end-tests/blob/d1b5a7a56ed1a12f64efd9a494089048cac90193/tests/helpers.py#L300-L313)

**Comment by viniarck** [comment]
> Results from [kytos-end-to-end-tests](https://github.com/amlight/kytos-end-to-end-tests/tree/master/tests) (e2e):
> 
> ```
> tests/test_e2e_01_kytos_startup.py ..                                    [  1%]
> tests/test_e2e_05_topology.py ..................                         [ 10%]
> tests/test_e2e_10_mef_eline.py .........Xss.....x....x.....              [ 24%]
> tests/test_e2e_11_mef_eline.py .....                                     [ 26%]
> tests/test_e2e_12_mef_eline.py .....xx.                                  [ 30%]
> tests/test_e2e_13_mef_eline.py .....xxXx......xxXx.XXxX.xxxx..x.E....... [ 51%]
> ...                                                                      [ 52%]
> tests/test_e2e_14_mef_eline.py x                                         [ 53%]
> tests/test_e2e_15_maintenance.py ........................                [ 65%]
> tests/test_e2e_20_flow_manager.py ................                       [ 73%]
> tests/test_e2e_21_flow_manager.py ..                                     [ 74%]
> tests/test_e2e_22_flow_manager.py ...............                        [ 81%]
> tests/test_e2e_23_flow_manager.py X......................                [ 93%]
> tests/test_e2e_30_of_lldp.py ....                                        [ 95%]
> tests/test_e2e_31_of_lldp.py ...                                         [ 96%]
> tests/test_e2e_32_of_lldp.py ...                                         [ 98%]
> tests/test_e2e_40_sdntrace.py X.x                                        [100%]
> ```
> 
> Summary: `170 passed, 2 skipped, 18 xfailed, 8 xpassed, 724 warnings, 1 error` Error: `Exception: Timeout while starting Kytos controller.` from [wait_controller_start()](https://github.com/amlight/kytos-end-to-end-tests/blob/d1b5a7a56ed1a12f64efd9a494089048cac90193/tests/helpers.py#L300-L313)

img

**Pull request merged**